File: plusDoesNotExistNumbers/src/main/kotlin/plusDoesNotExistNumbers.java
import java.util.Arrays;

class Solution {
    public int solution(int[] numbers) {
        int answer = -1;

        System.out.println(Arrays.toString(numbers));
        //Arrays.sort(numbers);
        int total = 45; // 0~9 의 합
        int sum = 0;
        for(int i = 0; i < numbers.length; i++)
            sum += numbers[i];
        answer = total - sum;
        return answer;

    }

}

File: plusDoesNotExistNumbers/src/main/kotlin/plusDoesNotExistNumbers.kt
class Solution {
    fun solution(numbers: IntArray): Int {
        var sum = 0
        for(i in 0..9){
            var isExist = false
            for(number in numbers){
                if(i == number){
                    isExist = true
                    break
                }
            }
            if(!isExist){
                sum += i
            }
        }
        return sum
    }
}


//========================================================

class Solution {
    fun solution(numbers: IntArray): Int {
        val exist = BooleanArray(10)
        for(num in numbers){
            exist[num] = true
        }
        var answer = 0
        for(i in 0..9){
            if(!exist[i]){
                answer += i
            }
        }
        return answer
    }
}
//==========================================================

class Solution {
    fun solution(numbers: IntArray): Int {
        return (0..9).filter{it !in numbers}.sum()
    }
}